index.html

```diff
@@ -0,0 +1,355 @@
+          <h2>About me</h2>
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+       <footer class="contact" id="contact">
+        <div class="contact-box">
+            <div class="contact-text-box padding-left-right">
+                <h2 class="contact-heading">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </h2>
+            </div>
+            <form class="contact-form" action="https://formspree.io/f/xayvglaa" method="post">
+                <fieldset class="contact-group">
+                    <label for="full_name"></label>
+                    <input type="text" id="full_name" name="user_full_name" placeholder="Full name" required />
```

Comment:
- [ ] You did an overall good job building your Name field here 👌. However, please kindly make sure that you include the fields First Name and Last Name instead of only Full Name to stick as much as possible to the Figma design as requested in the requirement below 👇🏽.
  > You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
  
 This is the Figma design 👇🏽
 
![figma](https://user-images.githubusercontent.com/116701485/233168393-e1217cf3-7051-4660-9668-b916a72c27b6.png)

This is your design 👇🏽 (almost there 😅)

![design](https://user-images.githubusercontent.com/116701485/233168521-a97351ae-43aa-4153-a7fd-dddc9b936a34.png)